### DISCUSSION TITLE
本地化方案

### DISCUSSION THREAD

**Discussion by Himalian** [JanSky520/nvimdoc_chs#6] [Ideas]
---
Update
这段时间想到的方案：
- 使用插件的形式安装文档
- 注册命令`:zh`，查询的时候把`:h vim.lsp`替换为`:zh vim.lsp`即可
- 中文文档保留与原文档相同的目录结构，仅替换内容
---

目前想到以下几种方案来启用中文文档

1. 直接替换:
    开发相对轻松，但使用、维护较麻烦

2. 使用插件:
    有一定开发难度，维护方便

3. 修改变量
## 使用插件
显然该方法更方便使用。存在的一些问题需要探讨：
 - 用什么语言开发？Lua/VimScript
 - 本地化的实现方法。可以替换文档、自定义新的``:help``命令或其他方法

[已有的Vim本地化方案](https://github.com/yianwillis/vimcdoc)
其他方法欢迎补充

**Comment by PelerYuan** [comment]
插件是不是可以看看 https://github.com/yianwillis/vimcdoc 中的能不能搬过来用

**Reply by Himalian** [reply]
我目前按照原文的符号排版，比较麻烦的是换行和字符画对齐
~~有自动化工具就好了~~

**Reply by Himalian** [reply]
[自动化工具](https://github.com/google/vimdoc)
